# HG changeset patch
# User zgu
# Date 1598885785 14400
#      Mon Aug 31 10:56:25 2020 -0400
# Node ID 44469e10dfe45ab8dd6fd2051048fc41132ca67a
# Parent  4b0aa85a95653f44cc45f2ec0571153017ebbf03
8252573: 8u: Windows build failed after 8222079 backport

diff --git a/src/share/vm/compiler/disassembler.cpp b/src/share/vm/compiler/disassembler.cpp
--- a/src/share/vm/compiler/disassembler.cpp
+++ b/src/share/vm/compiler/disassembler.cpp
@@ -251,7 +251,6 @@
   _output(output ? output : tty),
   _start(NULL),
   _end(NULL),
-  _option_buf(),
   _print_raw(0),
   // by default, output pc but not bytes:
   _print_pc(true),
